src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1799,6 +1799,16 @@
     /**
      * The {@code break} keyword.  The first child is an optional
      * identifier and the last child is a semicolon.
+     * <p>For example:</p>
+     * <pre>
+     *      break;
+     * </pre>
+     * <p>parses as:</p>
+     * <pre>
+     * +--LITERAL_BREAK -&gt; break
+     *      |
+     *      +--SEMI -&gt; ;
```

Comment:
Please remove `|`. No need for empty lines here.
Please replace `+` with ` to match the tool output:
```
    |       |       |       |--LITERAL_BREAK -> break [6:16]
    |       |       |       |   `--SEMI -> ; [6:21]
```